core/responsive/files/frontend.php

```diff
@@ -31,7 +32,8 @@ public function __construct( $file_name, $template_file = null ) {
 	 * @access public
 	 */
 	public function parse_content() {
-		$breakpoints = Plugin::$instance->breakpoints->get_active_config();
+		/** @var Breakpoint[] $breakpoints */
```

Comment:
When you fix the DOC you won't need this comment anymore

Reply:
Done.